### DISCUSSION TITLE
Reimagining graduation presentation in OLS

### DISCUSSION THREAD

**Discussion by malvikasharan** [open-life-science/open-life-science.github.io#852] [Ideas and brainstorms]
We are thinking about how else (beyond presentation) we can have our cohort members graduating.
- [ ] Art installation
- [ ] Public speaking
- [ ] Video submission
- [ ] Blog or storytelling
- [ ] Local public engagement
- [ ] Personal event they organise to talk about their project
- [ ] something else?

**Comment by Emidowojo** [comment]
If you're still thinking of innovative ways the cohort members can graduate, Blog or storytelling and video submissions are great ideas, especially in this post-pandemic period.

**Comment by SaraVilla** [comment]
@iramosp did you trial anything like this for Nebula? I remember that video submissions were available at certain point?

**Comment by SaraVilla** [comment]
closing as these options will be documented in the cohort handbook
